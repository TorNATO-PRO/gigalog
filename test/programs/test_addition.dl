sum(X, Y, Z) :- Z = X + Y.

sum(1, 2, 3).
sum(5, 7, 12).
sum(10, 20, 30).
