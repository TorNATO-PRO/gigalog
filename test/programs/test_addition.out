sum(1,2,3)
sum(10,20,30)
sum(5,7,12)
